src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +192,41 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str):
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await ImageRow.from_alias(session, alias)
+            if image_row is None:
+                raise UnknownImageReference
+
+            return validate_image_labels(image_row.labels)
+        except UnknownImageReference:
+            log.error(f"No images were found with alias: {alias}")
+        except Exception:
+            log.exception("An error occurred.")
+
+
+async def validate_image_canonical(cli_ctx, canonical: str, architecture: str):
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await session.scalar(
+                sa.select(ImageRow).where(
+                    (ImageRow.name == canonical) & (ImageRow.architecture == architecture)
+                )
+            )
+            if image_row is None:
+                raise UnknownImageReference(f"{canonical}:{architecture}")
```

Comment:
Although there is no standardized rule yet, but I think it would be better to notate arhictecture using a slash after the image tag as it's a commonly used convention.
```suggestion
                raise UnknownImageReference(f"{canonical}/{architecture}")
```